common-arcana.lic

```diff
@@ -236,4 +237,52 @@ module DRCA
       return parse_mana_message(mana_msg)
     end
   end
+
+  def cast_spells(settings, spells)
+    infuse_om(!settings.osrel_no_harness, settings.osrel_amount)
+    spells.each do |name, data|
+      next if DRSpells.active_spells[name] && (data['recast'].nil? || DRSpells.active_spells[name].to_i > data['recast'])
+      while mana < 40
+        echo('Waiting on mana...')
+        pause 15
+      end
+
+      if data['ritual']
+        ritual(data)
+      else
+        cast_spell(settings, data)
+      end
+    end
+  end
+
+
+  def cast_spell(settings, data)
+    if data['moon']
```

Comment:
I really kind of want this whole moon block moved into it's own method as a guard >.>